testdata/iceberg/tables/lineitem_partitioned/metadata/v1.metadata.json

```diff
@@ -0,0 +1,148 @@
+{
+  "format-version" : 2,
+  "table-uuid" : "d3224394-5406-4ba2-8abf-8c794a200c89",
+  "location" : "/Users/sean/Code/github.com/glaredb/glaredb/testdata/iceberg/tables/lineitem_partitioned",
```

Comment:
i think we may need to autogenerate these. the `generatd_iceberg` script fails locally as this is hardcoded to your filesystem.

Reply:
I accidentally used an expanded path in the script, pushing up a fix.